tests/test_standard_tags.py

```diff
@@ -109,11 +110,13 @@ def setup_url_with_sensitive_query_string(self):
         ]
 
     @missing_feature(
-        context.library in ["dotnet", "golang", "nodejs", "php", "ruby"],
+        context.library in ["golang", "nodejs", "php", "ruby"],
         reason="tracer did not yet implemented the new version of query parameters obfuscation regex",
     )
     @missing_feature(context.library < "java@1.21.0", reason="previous obfuscation regex")
     @irrelevant(context.library < "python@1.19", reason="python released the new version at 1.19.0")
+    # remove = of <= when PR https://github.com/DataDog/dd-trace-dotnet/pull/4504 is merged
+    @irrelevant(context.library <= "dotnet@2.40", reason="dotnet released the new version at 2.39.0")
```

Comment:
```suggestion
    @irrelevant(context.library < "dotnet@2.39", reason="dotnet released the new version at 2.39.0")
```

Reply:
we just had a release to 2.40 actually so it's 2.40 now, 
the reason I have the = is because my PR is not merged: if I merge it, I'll break the system tests, and if I merge this one, I'll break the tests, so for now I deactivate the new tests for 2.40 before the merge